README.md

```diff
@@ -1,97 +1,111 @@
 
 ## Directory structure
 
-`models` - Example neuron models in NestML format.
+`models` - Example neuron models in NESTML format.
 
-`pynestml` - The source code of PyNestML.
+`pynestml` - The source code of the PyNESTML toolchain.
 
 `tests` - A collection of tests for testing of the toolchain's behavior.
 
-`doc` - The documentation of the modeling language NestML as well as processing toolchain PyNestML.
+`doc` - The documentation of the modeling language NESTML as well as processing toolchain PyNESTML.
 
 `extras` - Miscellaneous development tools, editor syntax highlighting rules, etc.
 
 ## Installing NESTML
 
-### Requirements
+Please note that only Python 3 is supported. The instructions below assume that `python` is aliased to or refers to `python3`, and `pip` to `pip3`.
 
-In order to execute the language tool-chain, Python in version 2 or 3 is required.
-To ensure correct installation and resolving of dependencies, Python's package manager [pip](https://pip.pypa.io/en/stable/installing/), the distribution tool [setuptools](https://packaging.python.org/tutorials/installing-packages/) as well as the python headers (``python-dev`` on Ubuntu/Debian, ``python-devel`` on most other linux platforms) package are required and should be installed in advance. The setup file additionally installs the following components:
+### Installing the latest release from PyPI
 
-* [SymPy in the version >= 1.1.1](http://www.sympy.org/en/index.html)
-* [NumPy in the version >=1.8.2](http://www.numpy.org/)
-* [Antlr4 runtime environment in the version >= 4.7](https://github.com/antlr/antlr4/blob/master/doc/python-target.md)
+The easiest way to install NESTML is to use the [Python Package Index (PyPI)](https://pypi.org). This requires the Python package management system `pip` to be installed. In Ubuntu, Mint and Debian Linux you can install `pip` as follows:
 
-In the case that no 'enum' package is found, additionally, enum34 has to be updated by
 ```
-pip install --upgrade pip enum34
+sudo apt install python3-pip
 ```
 
-First of all, you should make sure that all required packages are installed.
-Requirements are stored in ``requirements.txt`` and can be installed in one step through pip by running
+NESTML can then be installed into your local user directory via:
+
 ```
-pip install -r requirements.txt
+pip install nestml --user
+```
+
+Optionally, you can run `pip` as root to install NESTML system-wide:
+
 ```
+sudo pip -H install nestml
+```
+
+### Installing the latest development version from GitHub
 
-### Installation
+To obtain the latest development version, clone directly from the master branch of the GitHub repository:
 
-Once all requirements are installed, a setup file is provided and can be run on Python 2 via 
 ```
-python2 setup.py install --user
+git clone https://github.com/nest/nestml
 ```
-Or for Python 3:
+
+First, install the requirements:
+
 ```
-python3 setup.py install --user
+cd nestml
+pip install -r requirements.txt
```

Comment:
is this really required?! shouldn't this be listed in the setup.py and be installed automatically...?